packages/tracker/tracker.js

```diff
@@ -196,6 +196,16 @@ Tracker.Computation = class Computation {
     this._onError = onError;
     this._recomputing = false;
 
+    /**
+     * @summary Forces autorun blocks to be executed in synchronous-looking order by storing the value autorun promise thus making it awaitable.
+     * @locus Client
```

Comment:
Is available in both server and client

Reply:
AFAIK the answer'd be yes. This is tracker package specific regardless of where it's gets executed.

Reply:
Yes, it's available on both.